scout/parse/variant/frequency.py

```diff
@@ -139,11 +139,11 @@ def parse_mei_frequencies(variant: cyvcf2.Variant) -> Dict:
 def parse_sv_frequency(variant, info_key):
     """Parse a SV frequency.
 
-    These has to be treated separately since some of them are not actually frequencies(float) but
-    occurances(int)
+    These have to be treated separately since some of them are not actually frequencies(float) but
+    occurences(int)
     """
     value = variant.INFO.get(info_key, 0)
-    if "AF" in info_key or "FRQ" in info_key:
+    if "AF" in info_key or "FRQ" in info_key or "Frq" in info_key:
```

Comment:
```suggestion
    if any(["AF","FRQ","Frq"]) in info_key:
```

Reply:
Yeah, something like this perhaps?
`if any(frq_string in info_key for frq_string in ["AF","FRQ","Frq"]):`

Reply:
The OR was fine anyway eh! 😄

Reply:
Thanks, I have a suggestion with `to_upper()` that I think should work. Let's see!

Reply:
> Ahem, `upper()`. Why don´t they just give the functions the same name in all languages.. 😁

Everybody wants to be original

Reply:
The devil is in the details. Perhaps that originality drive is the babel thing. Or it could just be that everyone intended to, only made slight mistakes of memory and typing that stuck. Thanks for the review!